src/main/java/org/folio/edge/rtac/utils/RtacUtils.java

```diff
@@ -0,0 +1,151 @@
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.apache.commons.collections4.CollectionUtils;
+import org.apache.commons.lang3.ArrayUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.edge.core.Constants;
+import org.folio.edge.core.utils.Mappers;
+import org.folio.edge.rtac.model.Instances;
+
+public final class RtacUtils {
+  public static final String FALLBACK_EMPTY_XML_RESPONSE = Mappers.XML_PROLOG + "\n<holdings/>";
+  public static final String FALLBACK_EMPTY_JSON_RESPONSE = (new JsonObject()
+      .put("holdings", new JsonObject())).toString();
+  public static final String SEPARATOR_COMMA = ",";
+
+  public static final String APPLICATION_XML = Constants.APPLICATION_XML;
+  public static final String TEXT_XML = Constants.TEXT_XML;
+  public static final String APPLICATION_JSON = Constants.APPLICATION_JSON;
+  public static final String ALL_WILDCARD = "*/*";
+
+  public static final String[] SUPPORTED_TYPES = new String[]{
+      APPLICATION_XML, TEXT_XML,
+      APPLICATION_JSON, ALL_WILDCARD};
+
+  public static final String[] XML_TYPES = new String[]{
+      APPLICATION_XML,
+      TEXT_XML,
+      ALL_WILDCARD};
+
+  private RtacUtils() {
+  }
+
+  private static String getPayload(boolean defaultXml) throws JsonProcessingException {
+    return (defaultXml) ? new Instances().toXml() : new Instances().toJson();
+  }
+
+  private static String getEmptyPayload(boolean defaultXml) {
+    return (defaultXml) ? FALLBACK_EMPTY_XML_RESPONSE : FALLBACK_EMPTY_JSON_RESPONSE;
+  }
+
+  private static String getContentType(boolean defaultXml) {
+    return (defaultXml) ? APPLICATION_XML : APPLICATION_JSON;
+  }
+
+  public static boolean hasAcceptHeader(HttpServerRequest request) {
+    return (request.headers().contains(ACCEPT) && StringUtils.isNotBlank(
+        getAcceptHeaderList(request).stream().collect(Collectors.joining())));
+  }
+
+  public static boolean hasRequestAnyOfAcceptTypes(HttpServerRequest request,
+      String... types) {
+    boolean result = false;
+
+    if (ArrayUtils.isNotEmpty(types)) {
+      var normalizedAcceptHeaders = getAcceptHeaderList(request);
+
+      result = normalizedAcceptHeaders.stream()
+          .anyMatch(reqMimeArrayEntry -> Stream.of(types)
+              .anyMatch(mimeType -> mimeType.equalsIgnoreCase(reqMimeArrayEntry)));
+    }
+    return result;
+  }
+
+  /**
+   * Check if there is accept header which lead to returning XML content (by business cases
+   * described in the: https://issues.folio.org/browse/EDGRTAC-16)
+   *
+   * @param request
+   * @return
+   */
+  public static boolean isXmlRequest(HttpServerRequest request) {
+    if (hasAcceptHeader(request)) {
+      return hasRequestAnyOfAcceptTypes(request, XML_TYPES);
```

Comment:
```suggestion
      return requestAccepts(request, XML_TYPES);
```

Reply:
Done

Reply:
No changes are appearing for me in github. Have these changes been pushed?

Reply:
I have pushed latest changes.